sqlglot/dialects/hive.py

```diff
@@ -109,6 +109,10 @@ def _array_sort_sql(self: generator.Generator, expression: exp.ArraySort) -> str
     return f"SORT_ARRAY({self.sql(expression, 'this')})"
 
 
+def notnullcolumnconstraint_sql(self, expression: exp.NotNullColumnConstraint) -> str:
+    return f"{'' if expression.args.get('allow_null') else 'NOT NULL'}"
```

Comment:
get rid of the f string it’s not necessary

Reply:
The f string prevents line wrap
<img width="735" alt="image" src="https://github.com/tobymao/sqlglot/assets/1122251/65934de5-54d6-45da-a8e5-9c1bf4a245f5">

Reply:
doesn't matter, please remove it

Reply:
Done